ImmuDB4Net/Utils.cs

```diff
@@ -211,7 +211,11 @@ public static string GenerateShortHash(string source)
         using (SHA256 sha1 = SHA256.Create())
         {
             var result = sha1.ComputeHash(Encoding.UTF8.GetBytes(source));
-            var hash = Convert.ToBase64String(result).ToUpper().Replace("=", "").Replace("/", "").Substring(0, 30);
```

Comment:
Can we use https://www.rfc-editor.org/rfc/rfc3548#page-6 for that? It's a base64 variant with slightly modified alphabet suitable for URLs and filenames